pilot/cmd/pilot-agent/status/ready/probe.go

```diff
@@ -42,16 +40,19 @@ func (p *Probe) Check() error {
 		return err
 	}
 
-	// Envoy has received some configuration, make sure that configuration has been received for
-	// all inbound ports.
-	if err := p.checkInboundConfigured(); err != nil {
-		return err
+	// Envoy has received some listener configuration, make sure that configuration has been received for
+	// any of the inbound ports.
+	if p.NodeType == model.Router {
```

Comment:
Maybe I am not caught up on this... but I thought split listener is configurable (but on by default)? Don't we still need the check for the case when its not split?

doesn't have to be this PR but we also probably don't need this for gateway now -- it was originally added for the case of listeners stuck warming but that is handled by the /status probe